avogadro/qtplugins/cartoons/cartoons.cpp

```diff
@@ -21,6 +21,7 @@
 #include <avogadro/rendering/spheregeometry.h>
 
 #include <functional>
+#include <iostream>
```

Comment:
Was this for debugging? It's good to minimize un-needed `#include` because they slow down compile time. (Which can be big for Avogadro.)